src/features/auth/store.ts

```diff
@@ -1,12 +1,19 @@
 import type { JsonRpcProvider } from '@ethersproject/providers';
 import create from 'zustand';
 
+import type { QueryManifest } from 'hooks/useApiBase';
```

Comment:
i don't love the dependency chain here... but i could probably resolve it by pulling fetchManifest into this feature